packages/taquito/src/contract/rpc-contract-provider.ts

```diff
@@ -207,4 +208,32 @@ export class RpcContractProvider extends OperationEmitter implements ContractPro
     const entrypoints = await this.rpc.getEntrypoints(address);
     return new Contract(address, script, this, entrypoints);
   }
+
+  async lambdaView(
+    lambdaContractOrAddress: Contract | string,
+    viewContractOrAddress: Contract | string,
+    viewMethod: string,
+    entrypointName?: string,
+    contractParameter?: MichelsonV1Expression
+  ): Promise<LambdaView> {
+    let lambdaContract;
+    let viewContract;
+    if (lambdaContractOrAddress instanceof Contract) {
```

Comment:
I believe the use of instanceof here can introduce a problem similar to what we faced in this issue: https://github.com/ecadlabs/taquito/pull/325

For example when a Contract is created by one taquito dependency and consumed by another instanceof will return false. This was the case with Truffle for example.